sympy/vector/tests/test_coordsysrect.py

```diff
@@ -463,19 +463,3 @@ def test_rotation_trans_equations():
            (-sin(q0) * c.y + cos(q0) * c.x, sin(q0) * c.x + cos(q0) * c.y, c.z)
     assert c._rotation_trans_equations(c._inverse_rotation_matrix(), c.base_scalars()) == \
            (sin(q0) * c.y + cos(q0) * c.x, -sin(q0) * c.x + cos(q0) * c.y, c.z)
-
-
-def test_translation_trans_equations():
```

Comment:
why did you remove this test?

Reply:
Because I removed a method `_set_inv_trans_equations` and these tests checks only this method